src/TestUtilities/appSettingsSW360IntegrationTest.json

```diff
@@ -1,15 +1,15 @@
 {
-  "FossologyURL": "",
+  "FossologyURL": "http://md2pdvnc.ad001.siemens.net:8096",
   "RemoveDevDependency": true,
   "SW360AuthTokenType": "Token",
   "SW360ProjectID": "0d0e23f6bccb4072be91b5a3462414ad",
   "SW360ProjectName": "Test",
-  "SW360URL": "",
+  "SW360URL": "http://md2pdvnc.ad001.siemens.net:8095",
```

Comment:
remove these urls and replace the place holders

Reply:
Removed